File: TP/JOURNEE_1/JAVA/ynov/Student.java
package ynov;

public class Student {
	
	private String name;
	
	private Integer age;
	
	public Student(String name, Integer age) {
		this.name = name;
		this.age = age;
	}
	
	public String getName() {
		return name;
	}
	
	public Integer getAge() {
		return age;
	}

	public String toString() {
		return "Student:name=" + name + ", age=" + age;
	}
}

File: TP/JOURNEE_1/C++/ynov/src/Student.cpp
#include "Student.h"
#include <string>
using namespace std;

Student::Student(string name, int age) {
    this->name = name;
    this->age = age;
}

string Student::getName() {
    return name;
}

int Student::getAge() {
    return age;
}

string Student::toString() {
    return "Student:name=" + name + "," + "age=" + to_string(age);
}